pytest_fixtures/component/provision_pxe.py

```diff
@@ -30,58 +30,85 @@ def module_provisioning_rhel_content(
     module_provisioning_sat,
     module_sca_manifest_org,
     module_lce_library,
-    module_default_org_view,
 ):
     """
     This fixture sets up kickstart repositories for a specific RHEL version
     that is specified in `request.param`.
     """
     sat = module_provisioning_sat.sat
     rhel_ver = request.param['rhel_version']
-    repo_names = [f'rhel{rhel_ver}']
-    if int(rhel_ver) > 7:
+    repo_names = []
+    if int(rhel_ver) == 7:
+        repo_names.append(f'rhel{rhel_ver}')
```

Comment:
Would we ever want to test anything < RHEL 7? Might be overkill but we could
```suggestion
    if int(rhel_ver) <= 7:
        repo_names.append(f'rhel{rhel_ver}')
```

Reply:
Extended Life Cycle Support for RHEL 6 lasts for another 18 months, and RHEL 6 is still a supported Satellite client version, so this is worth considering.

Reply:
Updated.